README.md

```diff
@@ -3,7 +3,10 @@
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.google.cloud.tools/jib-maven-plugin/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.google.cloud.tools/jib-maven-plugin)
 [![Gradle Plugin Portal](https://img.shields.io/badge/gradle%20plugin-v0.9.6-blue.svg)](https://plugins.gradle.org/plugin/com.google.cloud.tools.jib)
```

Comment:
Could maybe update this gradle version too?  (It's odd, but the maven badge remains a 0.9.6 — it appears Github is caching these images?)

Reply:
Oh yeah, we still need to update all the 0.9.6 occurrences to 0.9.7.

Reply:
Yea, I think this is the same problem: https://github.com/softwaremill/maven-badges/issues/5

Reply:
We might want to consider changing to use this:

[![Maven Central](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/google/cloud/tools/jib-maven-plugin/maven-metadata.xml.svg?colorB=007ec6)](https://maven-badges.herokuapp.com/maven-central/com.google.cloud.tools/jib-maven-plugin)

Reply:
I purged the camo cache for the Maven image with https://stackoverflow.com/questions/26898052/how-to-force-image-cache-update-in-readme-rst-on-github